config/routes.rb

```diff
@@ -5,6 +5,8 @@
     delete '/logout' => 'users/omniauth_callbacks#log_out'
   end
 
+  get '/login' => 'pages#login', as: :login
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.